packages/kaizen-tg/_templates/drupal-9-theme/new/.stylelintignore

```diff
@@ -2,3 +2,5 @@
 to: <%= h.src() %>/<%= h.changeCase.lower(name) %>/.stylelintignore
 ---
 /color/*.css
+dist/**/*.css
+*_external.css
```

Comment:
It's redundant, probably disabling `dist/**/*.css` will be already enough. Please test

Reply:
It is because of lint checking file on "Drupal/src" & storybook linter check files of external libraries css.